### DISCUSSION TITLE
Can I bundle the C# .net SDK with a project?

### DISCUSSION THREAD

**Discussion by cyraid** [dotnet/runtime#84076] [Q&A]
Am I allowed to ship/bundle a C# SDK along with my project? Windows C# SDK when on Windows, and Linux C# SDK when on Linux?

**Comment by cihatsolak** [comment]
Whether you can package the C# .NET SDK with your project depends on the SDK's license agreement. Open source licenses usually allow grouping, but sometimes private licenses have restrictions. It is necessary to pay attention to this. The C# .NET SDK is usually distributed under an open source license, so unpacking should generally be allowed. I still recommend checking certain license terms for eligibility.

**Reply by cyraid** [reply]
Yeah, was kinda hoping someone already looked into that and would know without me having to look. Wow, how lazy of me right? Haha

**Reply by cyraid** [reply]
Also thanks for the reply!
